src/index-sdk/uniV3Farm.ts

```diff
@@ -356,17 +355,17 @@ async function isTokenFromValidPool(
   return farm.pool?.toLowerCase() === nftPoolAddress?.toLowerCase()
 }
 
-export const getUpcomingFarms = () => {
-  return DpiEthRewards.farms.filter((farm: FarmData) => {
+export const getUpcomingFarms = (farm: V3Farm) => {
```

Comment:
This param could be named better, but I think I'm guilty of using it too 😂
`farm.farms` is definitely a code smell